mmime.c

```diff
@@ -336,7 +336,7 @@ print_header(char *line) {
 				s++;
 			if (linelen >= 78 - 13 - 4 ||
 			    (e-s < (78 - 13)/3 &&
-			     e-s >= (78 - linelen - 13)/3)) {
+			     e-s >= (int)(78 - linelen - 13)/3)) {
```

Comment:
Comparing l. 337, `(78 - linelen - 13)` cannot become negative here, i.e. the type of the operands is rather arbitrary.